src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1015,12 +1016,21 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
                     index: index
                 });
             });
+
+        } else if (errorResponse) {
```

Comment:
@ptewson-sfdo , the GE form error handling was changed to account to an error response not containing message and index, so that one error returned from the iframe error response can be displayed.